### PR TITLE
DIG-1220: replace setup.py with pyproject.toml

### PR DESCRIPTION
**Pull request by daisieh**
This shouldn't affect anything: Travis still works.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+[build-system]
+requires = ["setuptools >= 61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+version = "v1.1.0"
+name = "candigv2_authx"
+dependencies = [
```
Comment (OrdiNeu) [inline pyproject.toml:8]: Q: Does Snyk check these deps?

**Review by OrdiNeu — APPROVED** [review]
Looks good to me, though I'm unsure how to test

**Comment by mshadbolt** [comment]
wondering if we should take this opportunity to also try to be consistent and meet standards for package structure? e.g. the src structure here: https://www.pyopensci.org/python-package-guide/package-structure-code/python-package-structure.html#

**Comment by mshadbolt** [comment]
actually I was looking at the wrong branch, don't mind me, I see you have included a src directory ...

File: pyproject.toml (hunk 1 above)
Comment (daisieh) [reply pyproject.toml:8]: It's a little unclear to me, based on https://docs.snyk.io/scan-using-snyk/supported-languages-and-frameworks/python. I'm not sure it was checking setup.py before either, unless we'd told it to do that.

**Pull request merged**